### ISSUE TITLE
sokol_app: Command+Q quit event on macOS

### ISSUE THREAD

**Issue by TheSpydog** [floooh/sokol#265]
Would it be possible to add support for the macOS `Command+Q` keyboard shortcut such that sokol_app registers the command as a system "quit" event? Having to click the red button in the title bar just feels wrong to my muscle memory. 😜

(I'm brand new to sokol_app, so if this is already possible, I apologize! I didn't see anything in the documentation about it, so I thought I'd ask here.)

**Comment by floooh** [comment]
Yeah, this seems to require special handling on macOS, and hopefully shouldn't be too much work. I need to figure out how it works though.

**Comment by cschar** [comment]
for anyone else looking for a quick snippet

```
     // in event callback function

        if (ev->key_code == SAPP_KEYCODE_Q ){
            if(ev->modifiers == SAPP_MODIFIER_SUPER) {
                sapp_request_quit();
            }
            
            // regular q stuff
        }
```
